mahmud-shna/style.css

```diff
@@ -0,0 +1,59 @@
+
+body {  
```

Comment:
Don't try to use the tag selectors, always use class instead because class are reusable and easy to control